manticore/manticore.py

```diff
@@ -574,10 +528,11 @@ def _fork_state_callback(self, state, expression, values, policy):
             manticore_context['visited'] = manticore_visited.union(state_visited)
         state.context['visited_since_last_fork'] = set()
 
-        logger.info("Forking, about to store. (policy: %s, values: %s)", policy,
-                ', '.join('0x{:x}'.format(pc) for pc in values))
+        logger.debug("Forking, about to store. (policy: %s, values: %s)",
```

Comment:
can we have this message appear at verbosity=2 also? i realize this is somewhat troublesome to only log this message and not all of the other debug messages. one solution could be to introduce a custom log level in between info and debug, but i think an easier and pragmatic solution can simply be to move this logger call into the executor right before we do `self.publish('will_fork_state', ...)` with log level INFO. i also think it is more correct to emit this event from the executor since it really is more of an executor level event, rather than a higher level manticore event